STYLE_GUIDE.md

```diff
@@ -1,3 +1,49 @@
 # Fortran stdlib Style Guide
 
-This document will describe the code style to use.
+Adopting a consistent style can improve code legibility through the choice of good naming conventions.
+In addition, style checks will be run during CI to flag any severe non-conformance.
+This allows code review discussions to focus on semantics and substance rather than pedantry.
+Consistent whitespace usage, and not poluting line endigs with trailing white space makes `git diff`s considerably more legible.
```

Comment:
```suggestion
Consistent whitespace usage, and not polluting line endings with trailing white space makes `git diff`s considerably more legible.
```

Reply:
Speaking of typos, can I add https://github.com/reviewdog/action-misspell as a github action? It may need a new/different token setup. It creates really nice PR comments and would have caught this.